runtime/io.js

```diff
@@ -122,31 +130,31 @@ function caml_ml_open_descriptor_out (fd) {
     fd:fd,
     opened:true,
     out:true,
-    buffer:"",
+    buffer_curr:0,
+    buffer:new Uint8Array(65536),
```

Comment:
why 65536?

Reply:
I think took that number from the native runtime I think